collections/_dev/events.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Events"
+slug: "events"
+layout: devpage
+hidden: false
+---
+
+**Lesson 3: Understanding Events in Canvas Plugins**
```

Comment:
This whole events page is in draft still -- it's not bad but I haven't gone through to update it. Consider a placeholder for now!